content/blog/2018/10/2018-10-09-telemetry.adoc

```diff
@@ -43,8 +43,8 @@ These measures are in place to strike a balance between the need to understand h
 ## Improving Jenkins through real-world data
 
 We're already created our first trial.
-Jenkins 2.143 includes a trial to gather information about how how common it is for instances to use Java system properties to disable (parts of) security fixes.
-Every time we publish a security fix we're not completely certain is safe to apply for everyone, we add another of these options -- just in case.
+Jenkins 2.143 includes a trial to gather information about how common it is for instances to use Java system properties to disable (parts of) security fixes.
+Every time we publish a security fix we're not completely certain it is safe to apply for everyone, we add another of these options -- just in case.
```

Comment:
This one is actually correct IMO.

Reply:
Agreed.  There  might be clearer phrasing.

Reply:
Yeah, I am fine both ways